plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/TestIcebergRegisterTableProcedure.java

```diff
@@ -425,6 +437,59 @@ public void testRegisterTableWithInvalidMetadataFileName()
         }
     }
 
+    @Test
+    public void testRegisterHadoopTableAndRead()
+            throws IOException
+    {
+        // create a temporary table to generate data file
+        String tempTableName = "temp_table_" + randomNameSuffix();
+        assertUpdate("CREATE TABLE " + tempTableName + " (id INT, name VARCHAR)");
```

Comment:
Because you use `file-a.orc` filename, it would be helpful to make explicit use  the `format` table property here.